datadog/resource_datadog_synthetics_global_variable.go

```diff
@@ -156,8 +156,8 @@ func resourceDatadogSyntheticsGlobalVariableCreate(ctx context.Context, d *schem
 	apiInstances := providerConf.DatadogApiInstances
 	auth := providerConf.Auth
 
-	syntheticsGlobalVariable := buildSyntheticsGlobalVariableStruct(d)
-	createdSyntheticsGlobalVariable, httpResponse, err := apiInstances.GetSyntheticsApiV1().CreateGlobalVariable(auth, *syntheticsGlobalVariable)
+	syntheticsGlobalRequestVariable := buildSyntheticsGlobalVariableRequestStruct(d)
```

Comment:
Shouldn't this be `SyntheticsGlobalVariableRequest` instead ?

Reply:
shit, yes.

Reply:
Thanks for catching that 🙇